bowtie/tests/test_integration.py

```diff
@@ -2006,24 +2006,24 @@ async def test_info_pretty():
     )
     assert stdout == dedent(
         f"""\
-        name: "always_invalid"
-        language: "python"
-        version: "v1.0.0"
-        homepage: "https://bowtie.report/"
-        issues: "https://github.com/bowtie-json-schema/bowtie/issues"
-        language_version: "{platform.python_version()}"
-        os: "{platform.system()}"
-        os_version: "{platform.release()}"
-        source: "https://github.com/bowtie-json-schema/bowtie"
-        dialects: [
-          "https://json-schema.org/draft/2020-12/schema",
-          "https://json-schema.org/draft/2019-09/schema",
-          "http://json-schema.org/draft-07/schema#",
-          "http://json-schema.org/draft-06/schema#",
-          "http://json-schema.org/draft-04/schema#",
-          "http://json-schema.org/draft-03/schema#"
-        ]
-        """,
+        ╭──────────────────┬─────────────────────────────────────────────────────╮
+        │ name             │ always_invalid                                      │
+        │ language         │ python                                              │
+        │ version          │ v1.0.0                                              │
+        │ homepage         │ https://bowtie.report/                              │
+        │ issues           │ https://github.com/bowtie-json-schema/bowtie/issues │
+        │ language version │ {platform.python_version():51} │
+        │ os               │ {platform.system():51} │
+        │ os version       │ {platform.release():51} │
+        │ source           │ https://github.com/bowtie-json-schema/bowtie        │
+        │ dialects         │ https://json-schema.org/draft/2020-12/schema        │
```

Comment:
This seems like a slight improvement, so thanks! Though while we're here, I think it'd be nice to even further improve this by introducing some grouping (e.g. os and os version belong together, and language and language version belong together, and the 4 of those likely belong together in one parent group about versions, etc.)

It'd be nice to put some thought into that while you're looking at this (how we can group these fields in such a way that related information is next to each other).